modules/csg/csg_shape.h

```diff
@@ -51,6 +51,7 @@ class CSGShape3D : public GeometryInstance3D {
 	};
 
 private:
+	Ref<Material> default_material = memnew(StandardMaterial3D);
```

Comment:
I don't think this is how `Ref<>` types are instantiated. It's usually like `default_material.instantiate();`.